### DISCUSSION TITLE
How can I dynamically change output config?

### DISCUSSION THREAD

**Discussion by timsofteng** [MusicPlayerDaemon/MPD#1407] [General]
Hello. I have external USB audio interface which I use from time to time.
And I need but prefect stream rarely.
It's slightly annoying to comment parts of config when I need it and uncomment another parts.
How to handle this properly?
Maybe some "on the fly" config changing...

**Comment by Stephanowicz** [comment]
Hi,
I dunno Your setup but mpc should do the trick....
`mpc outputs| grep 'Output'` to get a list of configured outputs from `/etc/mpd.conf`
this assumes that the listed devices are configured in `/etc/alsa/conf.d/..`
Then You may switch to the desired output device with `mpc enable only`  plus the number of the device from the list

Cheers, Stephan
